llvm/tools/llvm-mca/Views/InstructionInfoView.cpp

```diff
@@ -14,13 +14,45 @@
 #include "Views/InstructionInfoView.h"
 #include "llvm/Support/FormattedStream.h"
 #include "llvm/Support/JSON.h"
+#include "llvm/Support/WithColor.h"
 
 namespace llvm {
 namespace mca {
 
+void InstructionInfoView::getComment(const MCInst &MCI,
+                                     std::string &CommentString) const {
+  StringRef s = MCI.getLoc().getPointer();
+  std::string InstrStr;
```

Comment:
judging from your later uses, I think we can use `StringRef` here. `std::string` is usually (much) more expensive than `StringRef`